### PR TITLE
merge MONDO:0024252 into MONDO:0014855

### LINKED ISSUE #7597: Intellectual developmental disorder, autosomal dominant 42 and Global developmental delay-neuro-ophthalmological abnormalities-seizures-intellectual disability syndrome
MONDO list the following terms as separate disorders but they are the same condition:

Global developmental delay-neuro-ophthalmological abnormalities-seizures-intellectual disability syndrome MONDO:0024252
Intellectual developmental disorder, autosomal dominant 42 MONDO:0014855

We may want to merge the terms and add the corresponding synonyms.

Of note, in NORD we have a report about this condition under the name "GNB1-Related Disorder" 

Thank you! Gioconda

### PR DESCRIPTION
**Pull request by katiermullen**
Closes #7597

merge MONDO:0024252 'global developmental delay-neuro-ophthalmological abnormalities-seizures-intellectual disability syndrome' into MONDO:0014855 'intellectual disability, autosomal dominant 42.'

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -343422,15 +343422,19 @@ id: MONDO:0014855
 name: intellectual disability, autosomal dominant 42
 def: "Any autosomal dominant intellectual disability in which the cause of the disease is a heterozygous mutation in the GNB1 gene. It is characterized by global developmental delay, intellectual disability, hypotonia, structural brain abnormalities, and seizures. Other less common findings include dystonia, visual impairment, behavior problems, growth delay, craniofacial defects, and genitourinary abnormalities in males." [https://clinicalgenome.org/affiliation/40006/, MONDO:patterns/disease_series_by_gene]
 subset: gard_rare {source="GARD:18501", source="MONDO:GARD"}
+subset: gard_rare {source="GARD:17893", source="MONDO:GARD"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:343425]: The source for this subset should be grouped.

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -343439,19 +343443,28 @@ synonym: "mental retardation, autosomal dominant 42" EXACT DEPRECATED [OMIM:6169
 synonym: "mental retardation, autosomal dominant type 42" EXACT DEPRECATED [MONDORULE:2, OMIM:616973]
 synonym: "MRD42" EXACT ABBREVIATION [DOID:0070072, OMIM:616973]
 xref: DOID:0070072 {source="MONDO:equivalentTo"}
+xref: GARD:17893 {source="MONDO:GARD"}
 xref: GARD:18501 {source="MONDO:GARD"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:343447]: there are 2 GARD x-ref. Can one of these be "preferred external"?

**Review by sabrinatoro — COMMENTED** [review]
I am updating the duplicated subset annotations, in addition, I will add the NORD label: "GNB1-Related Disorder"

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:343447]: @sabrinatoro GARD:18501 is the preferred external

Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:343447]: Never mind. These are not equivalent.

**Review by sabrinatoro — APPROVED** [review]
approving and merging

**Pull request merged**